src/components/marker/MarkerFullParcours.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useContext, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import * as _ from 'lodash';
+import { Marker } from 'react-leaflet';
+import { Icon } from 'leaflet';
+import { Dialog, DialogContent, DialogTitle } from '@material-ui/core';
+import Capsule from '../capsule/Capsule';
+import { ApiContext } from '../../contexts/ApiContext';
+
+const IconStyle = new Icon({
+  iconUrl: '/images/pin.png',
+  iconSize: [34, 34],
+});
+
+const MarkerExplore = () => {
+  const { course, capsules } = useContext(ApiContext);
+
+  const allCourses = [...course];
+  const allCaps = [...capsules];
+  const { id } = useParams();
+  const [open, setOpen] = useState(false);
+  const [scroll, setScroll] = useState('body');
+  const [currentPoi, setPoi] = useState(0);
+  const [currentCap, setCap] = useState(0);
+
+  const handleScroll = (scrollType) => () => {
+    setScroll(scrollType);
+  };
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+  const uniqueCapsuleById = _.uniqBy(allCaps, 'capsule_id');
+
+  console.log(id);
```

Comment:
Nope!

Reply:
I specifically search for them though ...